translations/tiled_ru.ts

```diff
@@ -1433,7 +1433,7 @@ Line %1, column %2</source>
         <location filename="../src/tiled/preferencesdialog.cpp" line="+60"/>
         <location line="+131"/>
         <source>Native</source>
-        <translation>Нативный</translation>
+        <translation>Системный</translation>
```

Comment:
Need to know the context of this string. Changing 'Native' to 'System' may be not a good choice if it is not meant.

Reply:
This is about the theme that tries to look like the system theme, which can also be called the "native" theme. Up to you guys what is the best way to convey this in Russian.

Reply:
There is another one - "По умолчанию" ("Default").

Reply:
New 'Системный' is better then.

Reply:
> There is another one - "По умолчанию" ("Default").

Yeah, I didn't go with "Default" because it's ambiguous which default is meant. In Tiled, the default theme on Windows and Linux is "Tiled Fusion".